example/static/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>landline-api test page</title>
+    <!-- I'm assuming here that Promises and XHR2 are avilable. Most notable, IE doesn't yet support promises, although they're currently in the technical preview. -->
```

Comment:
I think this is all right for now, since only developers will see this page. Eventually, we'll want to test on IE (especially if Landline's targeting the enterprise).